src/Test/Plutip/Internal/BotPlutusInterface/Run.hs

```diff
@@ -146,6 +148,8 @@ runContractWithLogLvl logLvl cEnv bpiWallet contract = do
         , pcCollateralSize = fromInteger defCollateralSize
         }
 
+    walletInfo = makeWalletInfo bpiWallet
```

Comment:
For this wallet `WalletInfo` is calculated twice from `BpiWallet` (second time in `withContractAs`). `makeWalletInfo` basically calculates pubkey hashes. `runContract` can be fed `WalletInfo` instead of `BpiWallet`, no problem. But I put it like that, to change less in `interactive-plutip` docs. Unsure..